content/kots-cli/admin-console/generate-manifests.md

```diff
@@ -14,17 +14,21 @@ kubectl kots admin-console generate-manifests [flags]
 
 This command supports the following flags:
 
-| Flag | Type | Description |
-|:-----|------|-------------|
-| `--rootdir` | string | root directory where the yaml will be written _(default `${HOME}` or `%USERPROFILE%`)_ |
-| `--shared-password` | string | shared password to use when deploying the admin console |
-| `--http-proxy` | string | sets HTTP_PROXY environment variable in all KOTS Admin Console components |
-| `--https-proxy` | string | sets HTTPS_PROXY environment variable in all KOTS Admin Console components |
-| `--no-proxy` | string | sets NO_PROXY environment variable in all KOTS Admin Console components |
-| `--with-minio` | bool | set to true to include a local minio instance to be used for storage _(default true)_ |
+| Flag                      | Type   | Description                                                                                                                                           |
+|:--------------------------|--------|-------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `--rootdir`               | string | Root directory where the YAML will be written _(default `${HOME}` or `%USERPROFILE%`)_                                                                |
+| `--namespace`             | string | Target namespace for the admin console                                                                                                                |
+| `--shared-password`       | string | Shared password to use when deploying the admin console                                                                                               |
+| `--http-proxy`            | string | Sets HTTP_PROXY environment variable in all KOTS admin console components                                                                             |
+| `--https-proxy`           | string | Sets HTTPS_PROXY environment variable in all KOTS admin console components                                                                            |
+| `--no-proxy`              | string | Sets NO_PROXY environment variable in all KOTS admin console components                                                                               |
+| `--with-minio`            | bool   | Set to true to include a local minio instance to be used for storage _(default true)_                                                                 |
+| `--minimal-rbac`          | bool   | Set to true to use the namespaced role and bindings instead of cluster-level permissions _(default false)_                                            |
+| `--additional-namespaces` | string | Comma delimited list to specify additional namespace(s) managed by KOTS outside where it is to be deployed. Ignored without with `--minimal-rbac=true` |
```

Comment:
A couple of questions:

"managed by KOTS outside " - what does "outside" mean here?

"Ignored without with" - I think you meant either without or with, but not both. Typo, methinks.